src/controllers/authController.ts

```diff
@@ -93,5 +95,61 @@ const login = async (req: Request, res: Response): Promise<void> => {
     sendInternalErrorResponse(res, err);
   }
 };
+const updatePassword = async (req: Request, res: Response): Promise<void> => {
```

Comment:
Let's shift this controller to the `userController` instead of keeping it in the `authController`.

Reply:
No Yves anything related to auth should stay in auth. Normal these are the controllers that goes in auth

1. Login
2. Logout
3. Forgot Password
4. Change Password
5. Reset password